src/components/constans/types.ts

```diff
@@ -0,0 +1,39 @@
+
+export interface SignupData {
+    name: string;
+    email: string;
+    password: string;
+    confirm: string;
+}
+
+export interface LoginData {
+    email: string;
+    password: string;
+}
+
+export interface IncomeData {
+    id: string;
+    monthIncome: number;
+    monthName: string;
+    year: number;
+}
+
+export interface ExpensesData {
+    [x: string]: any;
```

Comment:
I know this is annoying, but we need to try to avoid any if possible ;) Could you please try to fix it?